modules/telco-core-scalability.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-core-ref-components.adoc
+
+:_content-type: REFERENCE
+[id="telco-core-scalability_{context}"]
+= Scalability
+
+New in this release::
```

Comment:
I'm going to stop commenting on this one but, if the section isn't applicable, maybe it doesn't need to be in the doc.

Reply:
It is in the original and pretty sure I have to reflect that in conversion.